src/Internal/Wallet/Cip30Mock.purs

```diff
@@ -108,11 +108,16 @@ withCip30Mock (KeyWallet keyWallet) mock contract = do
     MockLode -> "LodeWallet"
 
 type Cip30Mock =
-  { getUsedAddresses :: Effect (Promise (Array String))
+  { getNetworkId :: Effect (Promise Int)
+  , getUtxos :: Effect (Promise (Array String))
   , getCollateral :: Effect (Promise (Array String))
-  , signTx :: String -> Promise String
   , getBalance :: Effect (Promise String)
-  , getUtxos :: Effect (Promise (Array String))
+  , getUsedAddresses :: Effect (Promise (Array String))
+  , getUnusedAddresses :: Effect (Promise (Array String))
+  , getChangeAddress :: Effect (Promise String)
+  , getRewardAddresses :: Effect (Promise (Array String))
+  , signTx :: String -> Promise String
+  , signData :: String -> Promise DataSignature
```

Comment:
It should accept two arguments (using `Fn2`) according to the spec, i.e. address and payload. Here it's only one.